pandas/core/array_algos/take.py

```diff
@@ -26,26 +26,12 @@
     from pandas.core.arrays.base import ExtensionArray
 
 
+# fmt: off
 @overload
-def take_nd(
-    arr: np.ndarray,
-    indexer,
-    axis: int = ...,
-    fill_value=...,
-    allow_fill: bool = ...,
-) -> np.ndarray:
-    ...
-
-
+def take_nd(arr: np.ndarray, indexer, axis: int = ..., fill_value=..., allow_fill: bool = ...) -> np.ndarray: ...  # noqa
 @overload
```

Comment:
can the noqa be made specific, i think `# noqa:F501,E704`

Reply:
done, thanks!